### PR TITLE
TF Updates

### PR DESCRIPTION
**Pull request by farizrahman4u**

### CODE REVIEW

File: nd4j/nd4j-tensorflow/src/main/java/org/nd4j/tensorflow/conversion/TensorDataType.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Skymind, Inc.
```
Comment (agibsonccc) [inline nd4j/nd4j-tensorflow/src/main/java/org/nd4j/tensorflow/conversion/TensorDataType.java:2]: Mind updating copyright to be 2019 Konduit on next line?

**Review by agibsonccc — DISMISSED** [review]

**Comment by agibsonccc** [comment]
@farizrahman4u  go ahead and finish this up so we can just move on here. There shouldn't be too much to do here.

**Comment by agibsonccc** [comment]
@farizrahman4u please make sure to run these tests on cpu/gpu: https://github.com/eclipse/deeplearning4j/tree/master/nd4j/nd4j-backends/nd4j-tests-tensorflow also look for new tests you need to backport from konduit-serving

**Review by AlexDBlack — CHANGES_REQUESTED** [review]
I've merged master to this branch and pushed it up

Testing this locally, I'm seeing:
(a) Dependency issue for GraphRunnerTest.testGraphRunnerFilePath and 2 others - `java.lang.NoClassDefFoundError: com/google/gson/GsonBuilder`
(b) JVM crash on GraphRunnerTest.testGraphRunnerSavedModel (note: test already existed before this PR) - https://gist.github.com/AlexDBlack/c52f4584122731cc8d44f5cf06c10cec
(c) Missing test resource: GraphRunnerTest.testGraphRunnerCast - `java.lang.IllegalStateException: Unable to read graph cast_float32_float64.pb`


I'll take a look at the dependency issue. Not sure where that cast graph is from.

**Comment by farizrahman4u** [comment]
@AlexDBlack adam uses generated graphs for dtype conversion - I forgot to check them in (`git add *.java`).

**Pull request closed without merging**